.gitignore

```diff
@@ -8,6 +8,7 @@
 /.bundle
 /Gemfile.lock
 /*.un~
+/.bye*
```

Comment:
Doesn't belong in this commit.

Also generally would belong in a global git ignore, not in this file. As with most of the things in this file.